.github/workflows/native-wsl.yml

```diff
@@ -0,0 +1,175 @@
+name: Native and WSL
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    defaults:
+        run:
+          shell: ${{ matrix.configuration == 'wsl' && 'wsl-bash {0}' || 'pwsh' }}
+    strategy:
+      matrix:
+        os: [windows-latest]
+        node-version: [16, 14, 12, 10]
```

Comment:
where's node 8 and below?

Reply:
I forgot to add 8 back. But 6 and 4 were commented out in the `appveyor.yml`.

Reply:
What versions should be tested? why not include 20, 19, 18, 17, 11, and 9 as well?

Reply:
Ideally we should - at least the evens.